### PR TITLE
3results page

### PR DESCRIPTION
**Pull request by trichoplax**

### CODE REVIEW

File: public/img/delete.svg [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Capa_1" x="0px" y="0px" width="512px" height="512px" viewBox="0 0 348.333 348.334" style="enable-background:new 0 0 348.333 348.334;" xml:space="preserve">
+<g>
+	<path d="M336.559,68.611L231.016,174.165l105.543,105.549c15.699,15.705,15.699,41.145,0,56.85   c-7.844,7.844-18.128,11.769-28.407,11.769c-10.296,0-20.581-3.919-28.419-11.769L174.167,231.003L68.609,336.563   c-7.843,7.844-18.128,11.769-28.416,11.769c-10.285,0-20.563-3.919-28.413-11.769c-15.699-15.698-15.699-41.139,0-56.85   l105.54-105.549L11.774,68.611c-15.699-15.699-15.699-41.145,0-56.844c15.696-15.687,41.127-15.687,56.829,0l105.563,105.554   L279.721,11.767c15.705-15.687,41.139-15.687,56.832,0C352.258,27.466,352.258,52.912,336.559,68.611z" fill="#716769"/>
+</g>
+<g>
+</g>
+<g>
```
Comment (jturner11) [inline public/img/delete.svg:10]: What do these spare <g> tags do? Should they be there?

File: public/js/main.js [hunk 2]
```diff
@@ -74,7 +49,85 @@ function change_img(section_item) {
...
+            var descriptionBox = this.parentNode.parentNode.querySelector(".description")
+            var close = descriptionBox.querySelector(".close_button")
+            if (this.getAttribute("id") % 2) {
+                $(descriptionBox).css("right", "30%")
+            } else {
+                $(descriptionBox).css("right", "40%")
+            }
+
+            close.addEventListener("click", function () {
+
```
Comment (jturner11) [inline public/js/main.js]: remove white space

File: public/js/resultsHandlebars.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/**
+ * implement Handlebars into our layout (results)
+ */
+function fillResultsTable(HBTemplate) {
+    var template = Handlebars.compile(HBTemplate)
+
+    Handlebars.registerHelper('if', function (v1, v2, options) {
+        if (v1 === v2) {
```
Comment (jturner11) [inline public/js/resultsHandlebars.js]: can you rename v1 & v2 to make them more semantic

File: public/js/roomHandlebars.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
+/**
+ * implement Handlebars into our layout
+ */
+function fillRoomTable(HBTemplate) {
+    var template = Handlebars.compile(HBTemplate)
+
+    Handlebars.registerHelper('if', function (v1, v2, options) {
+        if (v1 === v2) {
```
Comment (jturner11) [inline public/js/roomHandlebars.js]: can we make these more semantic

File: templates/index.phtml [hunk 5]
```diff
@@ -13,13 +13,39 @@
...
+            <img src="./img/logo_4.png" alt="Awesome Badger logo">
+        </a>
+        <h1>Awesome badger bookings</h1>
+    </div>
+    <div class="date_range_picker">
+        <form action="/" method="GET">
+            <div class="form_content">
+                <input type="text" id="from" name="from" placeholder="From...">
+                <input type="text" id="to" name="to" placeholder="To...">
+                <input type="submit" class="date_submit" value="Book">
```
Comment (jturner11) [inline templates/index.phtml]: change book to search

File: templates/index.phtml [hunk 6]
```diff
@@ -13,13 +13,39 @@
...
+            </div>
+        </form>
+        <div class="rooms_results"></div>
+    </div>
+
+    <div class="scroll">
+        <p>Checkout</p>
```
Comment (jturner11) [inline templates/index.phtml]: change checkout to 'take a look'

**Review by jturner11 — CHANGES_REQUESTED** [review]
we want to see a message if all rooms are booked

File: public/img/delete.svg (hunk 1 above)
Comment (bennlongg) [reply public/img/delete.svg:10]: its an svg file, so i feel inclined to leave it how it was downloaded

File: public/css/style.css [hunk 7]
```diff
@@ -361,4 +399,76 @@ body {
...
+    font-style: normal;
+    text-align: justify;
+}
+
+.room_card:nth-child(even) .description {
+    right: 70%;
+}
+
+.description img {
+
```
Comment (pdkoding) [inline public/css/style.css]: fix the empty line pls

**Review by pdkoding — CHANGES_REQUESTED** [review]
fix just one empty line pls

**Pull request merged**